libobs/data/color.effect

```diff
@@ -20,17 +20,17 @@ float3 srgb_nonlinear_to_linear(float3 v)
 
 float3 rec709_to_rec2020(float3 v)
 {
-	float r = dot(v, float3(0.6274040f, 0.3292820f, 0.0433136f));
-	float g = dot(v, float3(0.0690970f, 0.9195400f, 0.0113612f));
-	float b = dot(v, float3(0.0163916f, 0.0880132f, 0.8955950f));
+	float r = dot(v, float3(0.6274038959f, 0.3292830384f, 0.0433130657f));
+	float g = dot(v, float3(0.0690972894f, 0.9195403951f, 0.0113623156f));
+	float b = dot(v, float3(0.0163914389f, 0.0880133079f, 0.8955952532f));
```

Comment:
Could you use the whole numbers provided? Just let the shader compiler truncate to 32 bits.

Also remove the f suffix to match this file's style. Needed in C++ to denote 32-bit float instead of double, but unnecessary for shader.